Play travis fixed

aasdhajkshd opened the pull request:
Всё очень интересно, нужна практика в реальном "боевом" сценарии. 
По сути, мы реализовали скрытый механизм проверки наших файлов на пробелы, точки с запятой и другие возможные "орфо"... вот проверить стоит еще на табуляцию, как пробелы в python файлах (?). Не совсем понял, что именно исправлял \n и \r? Еще по-сути это как работает как dos2unix?
Проверка выполняется "нами" созданного некоторого скрипта прохождения которого, мы получаем штатно ответ и понимаем, сбойная версия или нет. chatGPT мне подсказывает, что это процесс CI/CD. Сам test.py - это презентация данного исполняемого кода, может быть как для HAProxy фермы - должны получить нужный ответ от сайта, типа healthcheck в docker'ах, возможно, и более сложные механизмы, как работает, полагаю, OpenSuSE Factory... Пример и задание прекрасно, но далее нужно самому уже изучать "габариты" данного подхода. Bravo.

aasdhajkshd committed:
Add PR template

aasdhajkshd committed:
Add play-travis test.py

aasdhajkshd committed:
Add auto-assign and run-test YAML

aasdhajkshd committed:
Fixing test.py

aasdhajkshd committed:
Fixing test.py

mrgreyves reviewed (changes requested):
Здравствуйте!
У вас где то потерялся Workflow для auto-assign
Нужно добавить

aasdhajkshd committed:
Add PR template (aasdhajkshd_infra / branch: play-travis)

Trying to solve Updates were rejected because the tip of your current branch is behind its remote couterpart.

aasdhajkshd committed:
Add PR template (again)

aasdhajkshd committed:
Changes to auto-assign.yml

aasdhajkshd committed:
Modified workflow/auto_assign.yml

aasdhajkshd committed:
Renamed workflow/auto_assign.yml to workflow/auto-assign.yml

mrgreyves reviewed (approved):
Здравствуйте!)
Вот теперь с вашим дз все круто =)

aasdhajkshd commented:
Play travis fixed #2
